paasta_tools/generate_all_deployments

```diff
@@ -6,18 +6,14 @@
 
 function delete_old_deployments {
     # Delete any deployments.json file that is older than an hour
-    find /nail/etc/services -iname deployments.json  -mmin +60 -delete
+    for service in $(comm -2 -3 <(find /nail/etc/services/ -maxdepth 1 -type d -printf %f\\n|tail -n +2|sort) <(paasta list|sort)); do
```

Comment:
I dare *don't* do this at all. These files don't hurt (afaik)
It isn't worth the risk of getting this bash wrong